### PR TITLE
Add a pointer to dstep

### PR DESCRIPTION
**Pull request by wilzbach**
Imho htod shouldn't be documented on dlang.org, because

- it's not part of the release archives
- it's closed source
- it's Windows-only
- there are more modern tools like dstep

As removing the documentation might be a bit drastic, I went with adding a warning.

### CODE REVIEW

**Comment by wilzbach** [comment]
Ping @CyberShadow and @andralex . So can we all agree that `htod` is super-outdated?
Maybe we can even remove it from the docs?

BTW funnily in the meantime Atila even posted his D <include> tool ...

**Comment by andralex** [comment]
@atilaneves include a link to your tool as well?

**Comment by CyberShadow** [comment]
Please link to the wiki instead: 
https://wiki.dlang.org/Bindings#Binding_generators

**Comment by andralex** [comment]
@CyberShadow good idea feel free to amend

File: htod.dd [hunk 1]
```diff
@@ -2,9 +2,8 @@ Ddoc
 
 $(D_S htod,
 
-$(MESSAGE_BOX orange, $(B Warning) - $(D htod) is no longer considered a core tool and maintained
-    by the same people. More modern tools like $(LINK2 https://github.com/jacob-carlborg/dstep, dstep)
-    might work better for your use case.)
+$(MESSAGE_BOX orange, $(B Warning) - `htod` is not developed actively. Alternatives include
```
Comment (CyberShadow) [inline htod.dd]: Please see $(LINK2 https://wiki.dlang.org/Bindings#Binding_generators, Binding generators on the D Wiki) for a list of alternatives.

**Comment by atilaneves** [comment]
I added dpp to the wiki.

**Comment by jacob-carlborg** [comment]
FYI htod is the first tool on that list in the wiki.

**Comment by wilzbach** [comment]
@jacob-carlborg I know, but logging to the wiki currently doesn't work for me:

image

Maybe someone else can update the list (i.e. making sure that dstep + dpp are at the top)?

**Comment by jacob-carlborg** [comment]
> Maybe someone else can update the list

Done.

**Pull request merged**